com.github.jeromerobert.pdfarranger.yaml

```diff
@@ -2,7 +2,7 @@ id: com.github.jeromerobert.pdfarranger
 runtime: org.gnome.Platform
 
 # Please also update 'flatpak-module-qpdf-pikepdf/generate-pikepdf' when changing runtime-version
-runtime-version: '45'
+runtime-version: '46'
```

Comment:
also the comment above was ignored

Reply:
true, this comment can be deleted / ignored.